### PR TITLE
Styleguide: Avoid the message rule attribute

### PR DESCRIPTION
**Pull request by j23414**
## Description of proposed changes

Conform to team styleguide: 

* https://docs.nextstrain.org/en/latest/reference/snakemake-style-guide.html#avoid-the-message-rule-attribute

## Checklist

- [x] Checks pass

### CODE REVIEW

File: ingest/workflow/snakemake_rules/trigger_rebuild.smk
Comment (victorlin) [inline ingest/workflow/snakemake_rules/trigger_rebuild.smk:1]: _(non-blocking)_

There's a bunch more in `phylogenetic/` too. Do you want to keep this PR scoped to `ingest/`, or make things consistent across the board?

**Review by victorlin — APPROVED** [review]

File: ingest/workflow/snakemake_rules/trigger_rebuild.smk
Comment (j23414) [reply ingest/workflow/snakemake_rules/trigger_rebuild.smk:1]: Oh good point! I can search and make it consistent across the board. Reduces the number of PRs.

Comment (j23414) [reply ingest/workflow/snakemake_rules/trigger_rebuild.smk:1]: Added the `phylogenetic/` message style fixes, with a minor grammatical correction. I didn't see any others across the repo, but let me know if you find any.

Comment (victorlin) [reply ingest/workflow/snakemake_rules/trigger_rebuild.smk:1]: Awesome! Looks good to me.

**Pull request merged**